src/components/modules/toolbar/index.ts

```diff
@@ -492,6 +515,16 @@ export default class Toolbar extends Module<ToolbarNodes> {
 
         this.moveAndOpen(data.block);
       });
+
+      /**
+       * Need to move toolbar on resize because we are now moving it to te left of a Block
+       * This is needed because max-width is removed from Toolbar content to accommodate different block layouts
+       */
+      this.readOnlyMutableListeners.on(window, 'resize', () => {
```

Comment:
we already have the window resize handler in the `Ui` module. Maybe you could use it?

Reply:
any update on that one?